File: src/latihanoop/Anggota.java
package latihanoop;

/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */

/**
 *
 * @author farha
 */

public class Anggota extends Person {
    
    public Anggota(int ID, String Nama, String KTP, String Alamat, String JenisKelamin) {
        super(ID, Nama, KTP, Alamat, JenisKelamin);
    }
    
    public String printInfoAnggota() {
        return
                "ID\t:" + getId()+ "\n" +
                "Nama\t:" + getNama()+ "\n" +
                "No.KTP\t:" + getKtp()+ "\n" +
                "ALamat\t:" + getAlamat()+ "\n" +
                "Jenis Kelamin\t:" + getJenisKelamin();
    }
    
}

File: app/src/main/java/lsp/project/borrowshelf/data/Anggota.kt
package lsp.project.borrowshelf.data

class Anggota(
  id: Int,
  nama: String,
  ktp: String,
  alamat: String,
  jenisKelamin: String,
) : Person(
  id,
  nama,
  ktp,
  alamat,
  jenisKelamin
) {
  fun printInfoAnggota() : String {
    return "ID\t: $id \n" +
      "Nama\t: $nama \n" +
      "No.KTP\t: $ktp \n" +
      "Alamat\t: $alamat \n" +
      "Jenis Kelamin\t: $jenisKelamin"
  }
}